superset/assets/src/components/RefreshChartOverlay.jsx

```diff
@@ -25,24 +25,33 @@ const propTypes = {
   height: PropTypes.number.isRequired,
   width: PropTypes.number.isRequired,
   onQuery: PropTypes.func,
+  showOverlay: PropTypes.bool,
+  overlayLabel: PropTypes.string,
 };
 
+const DEFAULT_OVERLAY_MESSAGE = 'Placeholder to show overlay message';
+
 class RefreshChartOverlay extends React.PureComponent {
   render() {
     return (
       <div
         style={{ height: this.props.height, width: this.props.width }}
         className="explore-chart-overlay"
       >
-        <div>
+        {!this.props.showOverlay && <div>
           <Button
             className="refresh-overlay-btn"
             onClick={this.props.onQuery}
             bsStyle="primary"
           >
             {t('Run Query')}
           </Button>
-        </div>
+        </div>}
+        {this.props.showOverlay && <div>
+          <form className="overlay-chart">
```

Comment:
Why we are using form tag ??

Reply:
Generally, `label` tags are used with `form` elements